preprocessors/autour/autour.py

```diff
@@ -86,6 +86,8 @@ def get_map_data():
             font=9&\
             pad=0"
 
+    api_request = ''.join(api_request.split())
```

Comment:
1) This should probably be assigned to @jaydeepsingh25 
2) Does this alter the content of the request? If the issue is whitespace, it would probably be sufficient to encode each space as `%20`.

Reply:
Ok sounds good I will change the assignee. No content changes for the request